src/components/TransactionResult.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  Alert,
+  AlertDescription,
+  AlertIcon,
+  AlertTitle,
+  Flex,
+} from "@chakra-ui/react";
+
+export enum ActionName {
```

Comment:
This component should not know or care about which strings we pass into it. I'd suggest moving enum to `src/types/types.ts` and give this enum a bit more telling name, current one is too generic imo